<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>484</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>727</width>
    <height>484</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(170, 170, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>461</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>461</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Lucida Console</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(170, 170, 255);
selection-background-color: rgb(255, 170, 255);
selection-color: rgb(255, 85, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="MainPage">
     <attribute name="title">
      <string>Main</string>
     </attribute>
     <widget class="QLineEdit" name="trackingNum">
      <property name="geometry">
       <rect>
        <x>185</x>
        <y>109</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(218, 208, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="trackingNumberLabel">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>104</y>
        <width>481</width>
        <height>41</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>481</width>
        <height>41</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 170, 255);

color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Tracking Number:</string>
      </property>
     </widget>
     <widget class="QListWidget" name="trackingDisp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>561</width>
        <height>221</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(85, 170, 255);</string>
      </property>
      <item>
       <property name="text">
        <string>Be sure to get any api_key  from EasyPost</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="trackButton">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>330</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="statusTip">
       <string>Track </string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 85, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Console&quot;;</string>
      </property>
      <property name="text">
       <string>Track!</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearAll">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>380</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 85, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Console&quot;;</string>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>191</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 85, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Console&quot;;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AmazonMws</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FedEx</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FedExMailview</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FedExSameDayCity</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FedexSmartPost</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>OnTrac</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UPS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>UPSIparcel</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>USPS</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="yourMomLabel">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>401</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Not Your Moms' Tracker</string>
      </property>
     </widget>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>260</y>
        <width>50</width>
        <height>64</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="etaText">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>152</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border-color: rgb(255, 238, 255);</string>
      </property>
      <property name="plainText">
       <string>Estimated Arrival Date and time </string>
      </property>
     </widget>
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>401</width>
        <height>37</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Lucida Console</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Package Tracker 2000:</string>
      </property>
     </widget>
     <zorder>title</zorder>
     <zorder>trackingNumberLabel</zorder>
     <zorder>trackingDisp</zorder>
     <zorder>trackButton</zorder>
     <zorder>clearAll</zorder>
     <zorder>comboBox</zorder>
     <zorder>yourMomLabel</zorder>
     <zorder>dial</zorder>
     <zorder>trackingNum</zorder>
     <zorder>etaText</zorder>
    </widget>
    <widget class="QWidget" name="SettingPage">
     <attribute name="title">
      <string>Setting</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>304</width>
        <height>76</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Enter API KEY FROM EASYPOST.COM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="apiKeyGet">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(218, 208, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QPushButton" name="enter">
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background:rgb(255, 85, 255);
color: rgb(255, 255, 255);
font: 9pt &quot;Lucida Console&quot;;</string>
          </property>
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuExit"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="statusTip">
    <string>New Tracking Number</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Saves current tracking</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="statusTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="statusTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
